language/tools/move-mv-llvm-compiler/src/stackless/translate.rs

```diff
@@ -900,6 +914,11 @@ impl<'mm, 'up> FunctionContext<'mm, 'up> {
 
         // Declare all the locals as allocas
         {
+            let func_name = &self.env.get_name_str();
+            debug!(target: "sbc", "Stackless local types for function {:?}", func_name);
+            for loc_ty in &fn_data.local_types {
+                debug!(target: "sbc", "{:?}", loc_ty);
+            }
```

Comment:
Similar to above, too much noise-to-signal. Like above, eliminate loop,:
`debug!(target: "sbc", "Stackless local types for function {:?}\n{:#?}", func_name, &fn_data.local_types);`

Reply:
Actually, this pretty print is bulkier, it produces something like:
```
[2023-06-13 03:21:30.581291667] move_mv_llvm_compiler::stackless::translate - language/tools/move-mv-llvm-compiler/src/stackless/translate.rs:922 [DEBUG] Stackless local types for function "moo"
[
    Struct(
        ModuleId(
            0,
        ),
        StructId(
            Symbol(
                4,
            ),
        ),
        [],
    ),
    Primitive(
        U32,
    ),
    Struct(
        ModuleId(
            0,
        ),
        StructId(
            Symbol(
                2,
            ),
        ),
        [],
    ),
    Struct(
        ModuleId(
            0,
        ),
        StructId(
            Symbol(
                4,
            ),
        ),
        [],
    ),
    Reference(
        false,
        Struct(
            ModuleId(
                0,
            ),
            StructId(
                Symbol(
                    4,
                ),
            ),
            [],
        ),
    ),
...
```
whereas printing per line is more compact:
```
[2023-06-13 03:16:59.720225087] move_mv_llvm_compiler::stackless::translate - language/tools/move-mv-llvm-compiler/src/stackless/translate.rs:920 [DEBUG] Struct(ModuleId(0), StructId(Symbol(4)), [])
[2023-06-13 03:16:59.720236779] move_mv_llvm_compiler::stackless::translate - language/tools/move-mv-llvm-compiler/src/stackless/translate.rs:920 [DEBUG] Primitive(U32)
[2023-06-13 03:16:59.720245265] move_mv_llvm_compiler::stackless::translate - language/tools/move-mv-llvm-compiler/src/stackless/translate.rs:920 [DEBUG] Struct(ModuleId(0), StructId(Symbol(2)), [])
[2023-06-13 03:16:59.720255856] move_mv_llvm_compiler::stackless::translate - language/tools/move-mv-llvm-compiler/src/stackless/translate.rs:920 [DEBUG] Struct(ModuleId(0), StructId(Symbol(4)), [])
[2023-06-13 03:16:59.720266386] move_mv_llvm_compiler::stackless::translate - language/tools/move-mv-llvm-compiler/src/stackless/translate.rs:920 [DEBUG] Reference(false, Struct(ModuleId(0), StructId(Symbol(4)), []))
...
```